moveit_msgs/ContactInformation[] contacts
